Tasks/AzureRmWebAppDeploymentV4/operations/KuduServiceUtility.ts

```diff
@@ -487,8 +509,7 @@ export class KuduServiceUtility {
             status : status,
             message : JSON.stringify(message),
             author : author,
-            deployer : 'VSTS',
-            details : buildOrReleaseUrl
+            deployer : 'VSTS'
```

Comment:
was the details parameter not used?

Reply:
No. It was not being added in the log as well, looks like this field is no more supported in kudu log and gets ignored.